SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+HLASM Language Support follows the Eclipse Vulnerability Reporting Policy. Vulnerabilities are tracked by the Eclipse security team, in cooperation with the HLASM Language Support project lead. Fixing vulnerabilities is taken care of by the HLASM Language Support project committers, with assistance and guidance of the security team.
+
+## Supported Versions
+
+HLASM Language Support supports security updates only for the latest version.
+
+## Reporting a Vulnerability
+
+We recommend that in case of suspected vulnerabilities you do not use the COBOL LS public issue tracker, but instead contact the Eclipse Security Team directly via security@eclipse.org.
```

Comment:
```suggestion
We recommend that in case of suspected vulnerabilities you do not use the HLASM LS public issue tracker, but instead contact the Eclipse Security Team directly via security@eclipse.org.
```

Reply:
thanks, fixed!